modules/cluster-logging-collector-log-forward-loki.adoc

```diff
@@ -0,0 +1,83 @@
+
+.Prerequisites
+
+* You must have an Loki logging system that is configured to receive the logging data using the specified protocol or format.
+
+.Procedure
+
+. Create a `ClusterLogForwarder` CR YAML file similar to the following:
++
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: ClusterLogForwarder
+metadata:
+  name: instance <1>
+  namespace: openshift-logging <2>
+spec:
+  outputs:
+   - name: loki-insecure <3>
+     type: "loki" <4>
+     url: http://loki.insecure.com:9200 <5>
+   - name: loki-secure
+     type: "loki"
+     url: https://loki.secure.com:9200 <6>
+     secret:
+        name: loki-secret <7>
+  pipelines:
+   - name: application-logs <8>
+     inputRefs: <9>
+     - application
+     - audit
+     outputRefs:
+     - loki-secure <10>
+     labels:
+       myLabel: "myValue" <11>
+   - name: infrastructure-audit-logs <12>
+     inputRefs:
+     - infrastructure
+     outputRefs:
+     - loki-insecure
+     labels:
+       logs: "audit-infra"
+----
+<1> The name of the `ClusterLogForwarder` CR must be `instance`.
+<2> The namespace for the `ClusterLogForwarder` CR must be `openshift-logging`.
+<3> Specify a name for the output.
+<4> Specify the `loki` type.
+<5> Specify the URL and port of the Loki system as a valid absolute URL. You can use the `http` (insecure) or `https` (secure HTTP) protocol. If the cluster-wide proxy using the CIDR annotation is enabled, the output must be a server name or FQDN, not an IP Address.
+<6> For a secure connection, you can specify an `https` or `http` URL that you authenticate by specifying a `secret`.
+<7> For an `https` prefix, specify the name of the secret required by the endpoint for TLS communication. The secret must exist in the `openshift-logging` project, and must have keys of: *tls.crt*, *tls.key*, and *ca-bundle.crt* that point to the respective certificates that they represent. Otherwise, for `http` and `https` prefixes, you can specify a secret that contains a username and password. For more information, see the following "Example: Setting secret that contains a username and password."
+<8> Optional: Specify a name for the pipeline.
+<9> Specify which log types to forward by using the pipeline: `application,` `infrastructure`, or `audit`.
+<10> Specify the name of the output to use when forwarding logs with this pipeline.
+<11> Optional: String. One or more labels to add to the logs.
+<12> Optional: Configure multiple outputs to forward logs to other external log aggregators of any supported type:
+** A name to describe the pipeline.
+** The `inputRefs` is the log type to forward by using the pipeline: `application,` `infrastructure`, or `audit`.
+** The `outputRefs` is the name of the output to use.
+** Optional: String. One or more labels to add to the logs.
+
+. Create the CR object:
++
+[source,terminal]
+----
```

Comment:
There are a couple of additional loki-specific properties documented here: 
    https://github.com/openshift/cluster-logging-operator/blob/master/pkg/apis/logging/v1/output_types.go#L181
Ideally we should be generating reference doc directly from `_types.go` but for now it's a cut/paste job.

Reply:
@alanconway  Please review the updated contents.